ckanext/geocat/harvester.py

```diff
@@ -119,6 +135,7 @@ def gather_stage(self, harvest_job):
             geocat_perma_link=self.config['geocat_perma_link_url'],
             geocat_perma_label=self.config['geocat_perma_link_label'],
             legal_basis_url=self.config['legal_basis_url'],
+            default_rights=self.config['rights'],
```

Comment:
This isn't the only place where we instantiate a GeoMetadataMapping object - it's also used in `commands.py` and in the tests. Please add the extra parameter in those places too.